src/ray/util/logging.cc

```diff
@@ -281,10 +302,12 @@ void RayLog::StartRayLog(const std::string &app_name,
       // logger.
       spdlog::drop(RayLog::GetLoggerName());
     }
+    auto filename = app_name_without_path + "_" + std::to_string(pid) + ".log";
     auto file_sink = std::make_shared<spdlog::sinks::rotating_file_sink_mt>(
-        JoinPaths(log_dir_, app_name_without_path + "_" + std::to_string(pid) + ".log"),
-        log_rotation_max_size_,
-        log_rotation_file_num_);
+        JoinPaths(log_dir_, filename), log_rotation_max_size_, log_rotation_file_num_);
```

Comment:
What do you mean "the rest of the PRs"? If you mean the test cases, that's because most test cases can't reach the size limit of a single log file. If not, please tell me the exact lines. 😀

Reply:
Yeah, I guess the fact log rotation never triggered in tests was my concern, i.e. not sure how the system will behave when log rotation kicks in, even though I feel it's 99.9% working :D

 Is there an easy way to actually trigger the log rotation in a test or a manual test run with some steps so that whoever works on it later could reproduce the behavior documented in this PR?